doc/schema/schema.md

```diff
@@ -68,16 +68,18 @@ Samples from the posterior distribution p(theta|y).
 ### `sample_stats`
 Information and diagnostics for each `posterior` sample, provided by the inference backend. It may vary depending on the algorithm used by the backend (i.e. an affine invariant sampler has no energy associated). The name convention used for `sample_stats` variables is the following:
 * `lp`: (unnormalized) log probability for sample
-* `step_size`
-* `step_size_bar`
+* `step_size`: the step size used for this sample
+* `step_size_bar`: the current best known step-size
 * `tune`: boolean variable indicating if the sampler is tuning or sampling
-* `depth`:
-* `tree_size`:
-* `mean_tree_accept`:
+* `depth`: HMC-NUTS only, the depth of the tree that was used to generate this sample
+* `tree_size`: HMC-NUTS only, the number of leafs of the sampling tree, when the sample was accepted
+* `n_leapfrog`: HMC-NUTS only, number of steps selected in each iteration to traverse the posterior
+* `mean_tree_accept`: HMC-NUTS only, the mean acceptance probability for the tree that generated this sample
 * `diverging`: HMC-NUTS only, boolean variable indicating divergent transitions
-* `energy`: HMC-NUTS only
-* `energy_error`
-* `max_energy_error`
+* `energy`: HMC-NUTS only, the energy at the point in phase-space where the sample was accepted
```

Comment:
Is it where the sample was accepted or proposed? e.g. if a transition was diverging, the initial point will be returned, but which energy is usually returned?

Reply:
I think in Stan initial point is not returned when transition is diverging. Point along the path between the initial and diverging point is selected.